### PR TITLE
Avatar

### PR DESCRIPTION
**Pull request by dev-ashishsethi**
Please give your valuable feedback

### CODE REVIEW

**Review by m26unkwn — APPROVED** [review]

**Comment by m26unkwn** [comment]
Dark them looks awsm.

**Review by dipen2000 — APPROVED** [review]

File: component.css [hunk 1]
```diff
@@ -129,6 +141,16 @@ a:visited{
 .text-heading{
     background-color: var(--white);
 }
+.social-links-group{
+    list-style: none;
+    padding: 0rem;
+}
+.social-links{
+    float: left;
```
Comment (rahulyadav139) [inline component.css:149]: use float is like the old days. We should use flexbox/grid/position instead of float for precise positioning.

**Review by rahulyadav139 — APPROVED** [review]

File: component/avatar/avatar.html [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+                    style="width: 955px; height: 336px; border:0; transform: scale(1); overflow:hidden;"
+                    sandbox="allow-scripts allow-same-origin">
+                </iframe>
```
Comment (dipen2000) [inline component/avatar/avatar.html:176]: instead of using carbon I suggest you use prims js and that way your website will be responsive as well.

**Review by dipen2000 — APPROVED** [review]
You need to make the website responsive yet, the website is not mobile-friendly. Dark color palette is good.
You are good to go.

**Pull request merged**